src/components/Inputs.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+
+import { StyleSheet, TextInput } from 'react-native';
+import { Searchbar } from 'react-native-paper';
+import { Colors } from 'src/constants/Colors';
+
+const styles = StyleSheet.create({
+  short: {
+    borderWidth: 2,
+    borderColor: Colors.Grey100,
+    borderRadius: 8,
+    color: Colors.Grey700,
+    backgroundColor: Colors.White,
+    minWidth: '100%',
+    padding: 10,
+  },
+  title: {
+    fontSize: 17,
+    fontFamily: 'Inter_800ExtraBold',
+  },
+  multiline: {
+    minWidth: '100%',
+    minHeight: 100,
+    backgroundColor: Colors.White,
+  },
+  search: {
+    borderRadius: 8,
+    marginBottom: 10,
+    minWidth: '100%',
+    elevation: 0,
+  },
+  outline: {
+    borderWidth: 1,
+    borderColor: Colors.Grey100,
+  },
+});
+
+export function ShortInput(props: React.ComponentProps<typeof TextInput>) {
+  return (
+    <TextInput
+      {...props}
+      style={[styles.short, props.style]}
+      placeholderTextColor={Colors.Grey300}
+    />
+  );
+}
+
```

Comment:
I feel like we're overoptimizing here, there's not that many instances currently where we need these components. We should only create them as necessary.